### PR TITLE
Fixes spacing betweens tags under the Tags section

### LINKED ISSUE #788: Proper spacing should be given in between the tags of photo detail page for mobile users
## Describe the bug
At present, there is not a proper alignment or space between the given tags of a particular photo.

## To Reproduce

1. Go to ccsearch-dev.creativecommons.org
2. Search for something
3. Use a browser to simulate a mobile breakpoint

## Expected behavior
Proper spacing should be given in between the tags.

## Screenshots

tags

## Desktop (please complete the following information)
 - OS: Chrome
 - Browser firefox

## Additional context
want to work on this, if CC staff thinks its a legit issue!

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
Fixes #788 by me

## Description
Made some changes for mobile users only, because changing for desktop users, doesn't make sense :)

## Screenshots
tags-fixed

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/styles/photodetails.scss [hunk 1]
```diff
@@ -64,3 +64,10 @@ label {
   height: 64px;
   display: block;
 }
+
+.button.tag {
+  /* Small only */
+  @media screen and (max-width: 39.9375em) {
```
Comment (brenoferreira) [inline src/styles/photodetails.scss:70]: You should instead add the `margin-vertical-normal` to the tag button [here](https://github.com/creativecommons/cccatalog-frontend/blob/develop/src/components/PhotoTags.vue#L6]

**Comment by abhisheknaiidu** [comment]
But @brenoferreira don't you think, it's now attaching side by side and also giving a lot of space in Mobile version

**Mobile :**
t1

**Desktop:**
t2

File: src/styles/photodetails.scss (hunk 1 above)
Comment (abhisheknaiidu) [reply src/styles/photodetails.scss]: Yup done :)

**Comment by abhisheknaiidu** [comment]
@brenoferreira Since its an issue in mobile version only, then why don't we just change the **height** of tags only taking Mobile breakpoints into consideration, I guess this will be better indeed for both the users, desktop and mobile!

**Pull request merged**